massgov/mayflower-react
# Pre push lint

This just adds the pre-push package and runs lint before a push goes through. It will only block pushing if there are hard errors found, warnings will be displayed but not block push.

To test:

* check out branch & npm install.
* create a branch and try to push it.
* verify you see lint warnings displayed.
* make a change that causes an actual lint error.
* try to push, verify the error shows and the push does not occur.
